modules/repository-s3/src/yamlRestTest/resources/rest-api-spec/test/repository_s3/10_basic.yml

```diff
@@ -1,16 +1,46 @@
 # Integration tests for repository-s3
 #
 "Module repository-s3 is loaded":
-    - skip:
-        reason: "contains is a newly added assertion"
-        features: contains
-    - do:
-        cluster.state: {}
+  - skip:
+      reason: "contains is a newly added assertion"
+      features: contains
+  - do:
+      cluster.state: { }
```

Comment:
nit: cleaning up stuff like whitespace in YAML tests is ok but I'd generally prefer them in a separate PR - it makes `git bisect` easier to use and avoids cluttering up `git annotate` output etc.

Reply:
I will keep it mind in next PR's. I applied project formatting to entire file, it's relatively small one.